<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>894</width>
    <height>653</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="Display" name="display_"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>894</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>General</string>
    </property>
    <addaction name="actionReset"/>
    <addaction name="actionLoad_Condif"/>
    <addaction name="actionSave_Config"/>
   </widget>
   <widget class="QMenu" name="menuSettings">
    <property name="title">
     <string>Settings</string>
    </property>
    <addaction name="actionSet_Kickstart"/>
   </widget>
   <widget class="QMenu" name="menuDebug">
    <property name="title">
     <string>Debug</string>
    </property>
    <addaction name="actionCPU"/>
    <addaction name="actionMemory"/>
    <addaction name="actionCopper"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuSettings"/>
   <addaction name="menuDebug"/>
  </widget>
  <widget class="StatusBarAmiga" name="statusbar">
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>16</height>
    </size>
   </property>
   <property name="baseSize">
    <size>
     <width>0</width>
     <height>16</height>
    </size>
   </property>
   <widget class="QLabel" name="led">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>0</y>
      <width>47</width>
      <height>16</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>32</width>
      <height>16</height>
     </size>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>Images/led_on.png</pixmap>
    </property>
   </widget>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="actionCPU">
   <property name="text">
    <string>Debug window</string>
   </property>
  </action>
  <action name="actionReset">
   <property name="text">
    <string>Reset</string>
   </property>
  </action>
  <action name="actionLoad_Condif">
   <property name="text">
    <string>Load Config</string>
   </property>
  </action>
  <action name="actionSave_Config">
   <property name="text">
    <string>Save Config</string>
   </property>
  </action>
  <action name="actionSet_Kickstart">
   <property name="text">
    <string>Set Kickstart</string>
   </property>
  </action>
  <action name="actionMemory">
   <property name="text">
    <string>Memory window</string>
   </property>
  </action>
  <action name="actionCopper">
   <property name="text">
    <string>Copper windows</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Display</class>
   <extends>QWidget</extends>
   <header>display.h</header>
   <slots>
    <signal>signal1()</signal>
    <slot>Update()</slot>
   </slots>
  </customwidget>
  <customwidget>
   <class>StatusBarAmiga</class>
   <extends>QStatusBar</extends>
   <header>statusbaramiga.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
